css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+* {
+  background-color: #4895ef;
+}
+
+h2,
+h3,
+p {
+  color: white;
+  font-family: 'Lato', sans-serif;
+}
+
+h3 {
+  height: 0;
+}
+
+.gray {
+  color: lightgray;
+  margin: 15px;
+  font-size: small;
+  line-height: 140%;
+}
+
+.heading {
+  width: 100%;
+  text-align: center;
+}
+
+.container {
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  padding: 8px;
+}
+
+.column {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+}
```

Comment:
This `column` class doesn't have a `width` or `flex-basis`. Make this a separate class, only `width` or `flex-basis` should be applied to `column` classes.